frontend/ui/data/css_tweaks.lua

```diff
@@ -185,6 +185,11 @@ img {
 }
             ]],
         },
+        {
+            id = "image_valign_middle";
+            title = _("Middle align images with text"),
```

Comment:
The natural way of saying this would be `Vertically center images with text` or `Vertically center align images with text (line)`. I suspect they chose "middle" in CSS mainly to avoid confusing with horizontal centering. I only suspected what this might even mean because I know CSS.

`Vertically middle align images with text` would do the trick as well, although I think that middle verges toward CSS jargon if not plain ungrammaticality.

For support, here's the relevant dialog from Word:

![2018-11-17 08_47_34-layout vertical alignment](https://user-images.githubusercontent.com/202757/48658648-a663c700-ea45-11e8-80f9-949e771ffab6.png)